src/components/Team/TeamData.jsx

```diff
@@ -0,0 +1,45 @@
+import team1 from './Assets/team-1.jpg'
+import team2 from './Assets/team-2.jpg'
+import team3 from './Assets/team-3.jpg'
+import team4 from './Assets/team-4.jpg'
+
+
+
+
+
+
+const TeamData=[
```

Comment:
it should be export const teamData instead of const TeamData.  keep this as .js file instead of .jsx. this is not a component. should not keep it as .jsx.